packages/module/src/Message/QuickStarts/QuickStartTileHeader.tsx

```diff
@@ -1,16 +1,16 @@
-import * as React from 'react';
 import { Button, Flex } from '@patternfly/react-core';
+import type { FunctionComponent, MouseEventHandler } from 'react';
 
 interface QuickStartTileHeaderProps {
   /** Name for the header */
   name: string;
   /** Id for the QuickStart */
   quickStartId?: string;
   /** Callback for when the name of the QuickStart is clicked */
-  onSelect: (e: React.FormEvent<HTMLInputElement> | React.MouseEvent<Element, MouseEvent>) => void;
+  onSelect: MouseEventHandler<HTMLButtonElement>;
```

Comment:
Does this cover the `FormEvent` as well? I'm not super familiar with the Handler types and having trouble finding the docs for it.